apps/stable_diffusion/web/ui/upscaler_ui.py

```diff
@@ -15,18 +13,17 @@
     get_custom_model_files,
     scheduler_list_cpu_only,
     predefined_upscaler_models,
-    cancel_sd,
```

Comment:
Why remove? Don't we want an option to cancel ?

Reply:
I'll put them back, I removed them because it didn't look like anything was using them.

Reply:
Hmm...We might want an option to cancel but at the moment `cancel_sd` isn't hooked up to the stop button on upscaler tab, this is the current code:

```python
        stop_batch.click(
            fn=None, cancels=[prompt_submit, neg_prompt_submit, generate_click]
        )
``` 

Which is probably why my editor flagged it as not used. Do you want me to change that on this PR or should I do it on another one?

Reply:
Import is back in as of my latest commit. Left the `stop_batch` click handler for now.